src/library.jl

```diff
@@ -25,6 +25,7 @@ function rwrap(pkg::ASCIIString,s::Symbol)
     m
 end
 
+"Import R Packages into Julia with or without alias. You can also use classic Python syntax to do this: `import *module-name* as *shorthand*`"
```

Comment:
seems a little strange to be referring to "classic python syntax" in a package to call R from Julia...